src/crimson/os/seastore/onode_manager/staged-fltree/stages/stage.h

```diff
@@ -1669,27 +1669,30 @@ struct staged {
         }
       }
     }
-    std::ostream& print(std::ostream& os, bool is_top) const {
+
+    template<typename OutputIt>
+    auto do_format_to(OutputIt out, bool is_top) const {
```

Comment:
need to drop the user of this method at lino 1733

Reply:
dropped.